assignments/objects.js

```diff
@@ -18,27 +18,69 @@ const example = {
 }
 
 // Write your intern objects here:
+const interns = {
+  mitzi: {
+    "id": 1,
+    "name": "Mitzi,F",
+    "email": "mmelloy0@psu.edu",
+    "gender": "F"
+  },
+
+  kennan: {
+    "id": 2,
+    "name": "Kennan,M",
+    "email": "kdiben1@tinypic.com",
+    "gender": "M",
+    speak() { console.log(`Hello my name is ${this.name}`) }
+  },
+  keven: {
+    "id": 3,
+    "name": "Keven,M",
+    "email": "kmummery2@wikimedia.org",
+    "gender": "M"
+  },
+  gannie: {
+    "id": 4,
+    "name": "Gannie,M",
+    "email": "gmartinson3@illinois.edu",
+    "gender": "M"
+  },
+  antonietta: {
+    "id": 5,
+    "name": "Antonietta,F",
+    "email": "adaine5@samsung.com",
+    "gender": "F",
+    multiplyNums(num1, num2) { return num1 * num2 }
+  }
+
+}
```

Comment:
I would go look at the solution code and see how their functions for the intern objects are written. As long as yours worked, they're not wrong, but they are written a little differently than how I'm used to seeing them.